fix: improve groups explanations (#4527)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/operations/access-control/index.md

You can centrally define role-based authorization within Kommander to control resource access on the management cluster and a set, or all, of the target clusters.
These resources are similar to Kubernetes RBAC but with crucial differences, and they make it possible to define the roles and role bindings once, and have them federated to clusters within a given scope.

Kommander has two concepts used to manage access control:

> - Kommander Roles: control access to resources on the management cluster.

Reviewer: I believe these are now changed in DKP in the UI

```suggestion
- DKP Roles: control access to resources on the management cluster.
```

Reviewer: This is how it is in the UI, but either way.

    name: oidc:engineering
```

> ## Types of Access Control Objects

Reviewer 2: I think this is all still relevant and should not be removed.

Author: Rodney, it is still relevant but it's already explained in the section above. This seemed to me a duplicate section. Will you verify?

    name: oidc:engineering
```

## Role Bindings

> Kommander role bindings, cluster role bindings, and project role bindings bind a group of users to any number of roles. All groups defined by [Identity Providers][groups] are present in the **Groups** tab at the global, workspace, or project level.

Reviewer 2: the "Groups" tab only exists at the global level for Identity Providers.

Everywhere else the tabs are Role Binding tabs. 

Role bindings or Cluster Role Binding tabs show a table with all the groups and all user to bind roles to those groups.

Reviewer: Yes - something like: 

```suggestion
Kommander role bindings, cluster role bindings, and project role bindings bind a group of users to any number of roles. All groups defined by [Identity Providers][groups] are present in the **Role Bindings** tab at the global and workspace level in the Access Control page, and in **Role Bindings** tab in the project level.
```

> Before you can create a Role Binding, ensure an administrator has created a [Group][groups]. A Kommander Group can contain one or several Identity Provider users, groups or both.

Reviewer 2: :+1:

> Before you can create a Role Binding, ensure an administrator has created a [Group][groups]. A Kommander Group can contain one or several Identity Provider users, groups or both.

Reviewer: 🎵 Oxford comma 🎵

```suggestion
Before you can create a Role Binding, ensure an administrator has created a [Group][groups]. A Kommander Group can contain one or several Identity Provider users, groups, or both.
```
